addons/sale_crm/models/crm_lead.py

```diff
@@ -87,6 +73,21 @@ def action_view_sale_order(self):
             action['res_id'] = orders.id
         return action
 
+    def _prepare_opportunity_quotation_context(self):
+        """ Prepares the context for a new quotation (sale.order) by sharing the values of common fields """
+        self.ensure_one()
```

Comment:
While we are at it, shoudln't we do the user / team thingy done in ``action_new_quotation`` also ?
```
        if self.team_id:
            action['context']['default_team_id'] = self.team_id.id,
        if self.user_id:
            action['context']['default_user_id'] = self.user_id.id
```